### ISSUE TITLE
Missing feature: Vpin value passing

### ISSUE THREAD

**Issue by DoganM95** [blynkkk/blynk-library#498]
Would be nice to have the functionality to pass the "When ..." Pin value over to the "set..." Pin. 

E.g. i have 2 Lights and also want a main switch, which switches both off simultaneously.

![Screenshot_20200916-151259.png](https://user-images.githubusercontent.com/38842553/93342096-3421c080-f82f-11ea-9606-6b63850e40a2.png)

This worked by creating 4 if...thens in eventor.

![Screenshot_20200916-151402.png](https://user-images.githubusercontent.com/38842553/93342190-54517f80-f82f-11ea-9072-acbf9bdc5a04.png)

But if i want to set the main brightness (0-1023) to each light, i would have to create 2x1024 cases..

![Screenshot_20200916-151735.png](https://user-images.githubusercontent.com/38842553/93342646-dcd02000-f82f-11ea-8357-21f10293a215.png)

To sum it up, the eventor needs one more function to pass the Source pins value to the destination on change.

![Screenshot_20200916-150304~2.png](https://user-images.githubusercontent.com/38842553/93343021-4c460f80-f830-11ea-8ab6-e52fbb5c02ca.png)

**Comment by vshymanskyy** [comment]
Hi! You can submit your ideas to https://portal.productboard.com/blynk/1-blynk-roadmap/
(and link to this issue). We do not accept such requests in this repository (this is firmware-related).
